project/.php_cs.dist

```diff
@@ -49,12 +48,14 @@ if (\PHP_VERSION_ID >= 50600) {
 }
 if (\PHP_VERSION_ID >= 70000) {
     $rules = array_merge($rules, [
+        '@PHP70Migration' => true,
         '@PHP70Migration:risky' => true,
         '@PHPUnit60Migration:risky' => true,
     ]);
 }
 if (\PHP_VERSION_ID >= 70100) {
     $rules = array_merge($rules, [
+        '@PHP71Migration' => true,
```

Comment:
This one is causing travis to fail because `const` does not have visibility defined. Do I change dev-kit code also or remove this line?

Reply:
Done